### DISCUSSION TITLE
[Enhancement] [radio] el-radio 高级功能 可以取消选择

### DISCUSSION THREAD

**Discussion by Mt-Youya** [element-plus/element-plus#14509] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `radio`

## Description

需要单选取消功能 , 类似 semi degsign 的 单选 高级模式 https://semi.design/zh-CN/input/radio

![1697004564415](https://github.com/element-plus/element-plus/assets/57708498/dcd0060b-6744-42ce-9235-0c57f730162e)

**Comment by xxholly32** [comment]
确实，不过原生的radio，也是这样的效果，其实用起来是比较别扭

这个也是一种取舍吧，不过通过点击事件判断也可以实现这个效果，element-plus只是不选择封装进去吧

**Reply by Mt-Youya** [reply]
是的,但是这个功能其实挺常见的,不算冷门. 我这里其实也加了事件,但是因为表单项很多,导致每一个radio都要添加, 最终添加的click事件会很多

**Comment by tolking** [comment]
原生行为就是这样，单选无法取消，多选可以取消。我觉得不应该破坏这种行为

可以使用 ElCheckbox，或者自定义组件
